client/menu/menu.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <link rel="stylesheet" href="../../third_party/codemirror-5.19.0/lib/material.css">
+    <link rel="stylesheet" href="menu.css">
+    <script src="../../third_party/angular-1.6.1/angular.min.js"></script>
+  </head>
+  <body>
+    <div class="tie-menu-question-list-wrapper">
```

Comment:
So can I do it without a JS file or do you want me to actually pull the data from the assets?